src/Contracts/Reporter.php

```diff
@@ -43,9 +43,9 @@ public function rawNotification(callable $callable): array;
     public function checkin(string $key): void;
 
     /**
-     * @param  int|string  $key
-     * @param  int|string  $value
-     * @return void
+     * @param  int|string|array  $key
+     * @param  int|string|array|null  $value
```

Comment:
Also, I think we should add descriptions to these parameters so it's clearer how they can be used. See the example in my earlier comment.

Reply:
Ah, just noticed you added the description on the implementation. I'm thinking it should be on the contract though.🤔

Reply:
Done, I moved the doc to the interface and replaced with inheritdoc, like the other methods.

I have not added the descriptions per parameter though, i don't think it's very clear because the description for $value is actually mostly a description for $key. Maybe it's just an issue of phrasing.

Reply:
Yeah, yours is better.